web/js/components/ModalStyle.jsx

```diff
@@ -21,7 +21,7 @@ export const MODAL_STYLE = {
     outline                    : 'none',
     padding                    : '7px',
     color                      : 'white',
-    height                     : '26em'
+    // height                     : '26em'
```

Comment:
why comment instead of delete?

Reply:
ended up removing after resolving merge conflicts